docs/Parameters.md

```diff
@@ -54,8 +54,8 @@ The parameter format is `key1=value1 key2=value2 ... ` . And parameters can be s
   * support multi validation data, separate by `,`
 * `num_iterations`, default=`100`, type=int, alias=`num_iteration`,`num_tree`,`num_trees`,`num_round`,`num_rounds`
   * number of boosting iterations
-  * note: `num_tree` here equal with `num_iterations`. For multi-class, it actually learns `num_class * num_iterations` trees.
-  * note: For python/R package, cannot use this parameters to control number of iterations.
+  * note: For python/R package, **this parameters is ignored**.
+  * note: internally, LightGBM constructs `num_class * num_iterations` trees for `multiclass` problems. Neither of aliases of this parameter directly control number of actually constructed trees.
```

Comment:
@Laurae2 

1. > rephrase as the "number of boosting iterations" and not only the "number of iterations" for less ambiguity.

   Do you suggest to change `... cannot use this parameters to control number of iterations` to `... cannot use this parameters to control number of boosting iterations`? 
I might be misunderstanding something, but that's the line I suggest to completely remove (and replace it with my version). Effectively, there's no "iterations" word in my version at all, so I'm not sure how to integrate your suggestion in it. Maybe you could provide the full text of what you suggest?

2. > In addition, you can remove the sentence "Neither of aliases of this parameter directly control number of actually constructed trees." (it is actually wrong)
    - Do you mean that some aliases behave differently? As far as I can see from [here](https://github.com/Microsoft/LightGBM/blob/c66f7415e712be305e0e39c95132e95859b40640/include/LightGBM/config.h#L391-L395) they all are equal.
    - Do you mean "they all allow _direct_ control"? My idea was that if `num_class` is involved, it means that the control is "indirect". The original version of this point was explicitly mentioning one of the aliases, the `num_tree`. AFAIU, this was to highlight that although it sounds like it directly controls number of trees, it actually doesn't. This my sentence was aiming to replace it. But I agree that it's OK to just remove it - it is already mentioned that it's just an alias and it should be enough.